taf/src/main/java/com/epam/gomel/tat/page/YandexDiskCreateFolderPage.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.gomel.tat.page;
+
+import com.github.javafaker.Faker;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class YandexDiskCreateFolderPage extends AbstractPage {
+
+    private static final By FOLDER_NAME_BOX = By.xpath("//form[@class='rename-dialog__rename-form']//input");
+    private static final By FOLDER_SAVE = By.xpath("//button[contains(.,'Сохранить')]");
+    private static final By CREATED_FOLDER = By.xpath("//div[text()='Вы создали папку ']");
```

Comment:
What will happen if a user uses Belarus or English on his browser?

Reply:
I changed the browser language to English. The page of Yandex Disk still loads in Russian. Tests will be correct.